addons/payment_ingenico/data/payment_acquirer_data.xml

```diff
@@ -11,5 +11,10 @@
             <field name="registration_view_template_id" ref="ogone_s2s_form"/>
         </record>
 
+        <record model="ir.config_parameter" id="payment_ingenico_hash_function" forcecreate="False">
+            <field name="key">payment_ingenico.hash_function</field>
```

Comment:
Renaming `payment_ogone` to `payment_ingenico` was a mistake. It has been reverted in 15.0 so better use "Ogone" for stored data ;)
```suggestion
        <record model="ir.config_parameter" id="payment_ogone_hash_function" forcecreate="False">
            <field name="key">payment_ogone.hash_function</field>
```

Reply:
@fdamhaut  Same goes for the XMLID

Reply:
Oops